index.js

```diff
@@ -0,0 +1,58 @@
+import Book from './modules1/book.js';
+import Store from './modules1/store.js';
+import ShowBooks from './modules1/showBooks.js';
```

Comment:
* The issue still persists the modules folder is not accessible because its a submodule, Kindly make modules a r normal folder so that its content can be accessible [You can go through this link for other solutions ](https://github.com/microsoft/vscode/issues/7829)

https://github.com/AlyAtalla/Awesome-books/assets/52497246/94fc4083-0514-45a5-9081-bd74ec920d22